### DISCUSSION TITLE
Adonisjs Inertia

### DISCUSSION THREAD

**Discussion by timesignature** [adonisjs/core#4472] [Help]
after building adonisjs, im getting the following error. Failed to load url inertia/app/ssr.tsx (resolved id: inertia/app/ssr.tsx). Does the file exist?, can anyone help please

**Comment by RomainLanz** [comment]
We have already answered your question.

> Please set NODE_ENV=production when running your application in production

https://github.com/adonisjs/core/discussions/4468#discussioncomment-8869985
